src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -338,6 +337,7 @@ static std::vector<SenderThreadParams*>* buildSenderParamsCustom
     params->renderFormat     = renderFormatToString(renderFormat);
     params->fiwareCorrelator = fiwareCorrelator;
     params->extraHeaders     = headers;
+    params->registration     = false;
```

Comment:
What about inventing an `enum RequestFamily`, with values like NormalRequest, AdminRequest, and RegistrationRequest, and using a variable of that type ?